regenerateStdlibForBrowser.js

```diff
@@ -21,7 +21,7 @@ try {
   console.log(`
 ** There's an error while reading ${stdlibDir}. Is this your first time setting things up?
 You need to copy the BuckleScript standard library files from here:
-https://github.com/BuckleScript/bucklescript/tree/d21cdfbbe61afb0ae6b79735765de949409947ea/lib/js
+https://github.com/BuckleScript/bucklescript-playground/tree/master/stdlib
```

Comment:
Humm, we're using the commonjs version; this is the amd one though? How does this work?

Reply:
Oh, well it doesn't look like it actually does. I've just been testing with code that didn't need any runtime. If it does need runtime, I get "define is not defined". Hold on, I'll fix.